mini-pools-ios/Models/ContributionModel.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ContributionModel.swift
+//  mini-pools-ios
+//
+//  Created by Kaplún, Sebi on 12/20/17.
+//  Copyright © 2017 Kaplún, Sebi. All rights reserved.
+//
+
+import UIKit
+import SwiftyJSON
+
+struct ContributionModel {
+    var id: Int
+    var amountValue: Int
+    var amountCurrency: String
+    var note: String?
+    var extra: String?
+    var createdAt: Date
+    var updatedAt: Date
+    var contributor : PersonModel
+    
+    init(dictionary : JSON ) {
```

Comment:
while our `SwiftLint` is not fully integrated yet in Venice, we tend to force this convention:
`init(dictionary: JSON)`

No need to fix it throughout the project now, just FYI

Reply:
👍